### DISCUSSION TITLE
When do image optimization usage limits reset for Hobby accounts?

### DISCUSSION THREAD

**Discussion by jasonwebb** [vercel/community#2425] [Help] [answered]
### Summary

I recently deployed a small portfolio site with a few hundred remote images that use the Next.js `<Image />` component to do image optimizations, and apparently I hit the 1000 image usage limit for my Hobby plan in just a couple days (oops?). I'm thinking I'll need to disable Next.js optimization for now and experiment with using some kind of third-party image CDN + optimization service with a custom loader, but in the short term I was curious if that 1000 image usage limit ever resets or if it is a lifetime limit across all projects?

From [the docs](https://vercel.com/docs/concepts/image-optimization/limits-and-pricing) it sounds like the usage limit for Pro (and higher) plans resets with each billing cycle, but I wasn't sure if the Hobby plan has such a "cycle" or if anything resets after some time. In my Settings -> Billing page I see a "current period" date range that ends on today's date, but I had a hunch that this always displays the last 30 days from the current day (?).

![image](https://github.com/vercel/community/assets/422783/8d34d7e9-6207-4f4f-a9f4-c3c5008d6f18)

**Comment by swarnava** [comment, marked as answer]
Hi,

Our Hobby plan can be utilized in a Production environment. However, please bear in mind that it's not intended to use it commercially for profit.

This usage is governed by our Fair Usage Policy and we strongly recommend all users to familiarize themselves with it to avoid any misunderstandings. You can find more details at https://vercel.com/docs/concepts/limits/fair-use-policy

if you have any questions or need any clarification, please do not hesitate to reach out to us.

**Reply by jasonwebb** [reply]
Got it. I'll hold off on doing a one-time reset on my account until I've sorted out custom loader with a third-party CDN service, since I think I'd just hit the limit again pretty quickly given my portfolio contains a good number of images. 

To be clear, this is just [my own small personal portfolio site](https://jasonwebb-io-next.vercel.app/), not a commercial project. I'm not sure if I understand what you mean by "experimental" and "production", but I don't see my use case violating anything in the Fair Use Policy. It's actually meeting most of the examples of Fair Use, and none of the examples of Never Fair Use. The word "production" is [only used once in the policy](https://vercel.com/docs/concepts/limits/fair-use-policy#:~:text=part%20of%20the-,production,-of%20the%20project) referring to the production (as in "creation") of a commercial project. My project fits the description of the Hobby plan on the Pricing page as a ["personal or non-commercial project"](https://vercel.com/pricing#:~:text=For%20personal%20or%20non%2Dcommercial%20projects.)

**Reply by danlevison** [reply]
I did the same thing :/ would I be able to get one-time-reset on my account as I won't be re-deploying my personal image heavy site until I find a solution for the images.
